products/index.html

```diff
@@ -30,66 +30,63 @@ <h1 class="header__title">
-          <div class="items-list clearfix">
-            <img class="image" src="../images/comp_p1.png" alt="product1">
-            <h3 class="product-name">COMP Powder LC v.1.1</h3>
-            <p class="desc">A slightly sweet rich milk flavor that is delicious
-              even at carb 4.9 g / 400 kcal.</p>
-            <p class="desc">It is a nutritional food of a carbohydrate
-              adjustment model that is packed with COMP's attention to detail.</p>
-            <button class="button">
-              <i class="fas fa-shopping-cart"></i> BUY COMP Powder LC
-            </button>
-          </div>
-          <div class="items-list clearfix">
-            <img class="image" src="../images/comp_p2.png" alt="product2">
-            <h3 class="product-name">COMP Powder TB v5.1</h3>
-            <p class="desc">A slightly sweet rich milk flavor that is delicious
-              even at carb 4.9 g / 400 kcal.</p>
-            <p class="desc">It is a nutritional food of a carbohydrate
-              adjustment model that is packed with COMP's attention to detail.</p>
-            <button class="button">
-              <i class="fas fa-shopping-cart"></i> BUY COMP Powder TB
-            </button>
-          </div>
-          <div class="items-list clearfix">
-            <img class="image" src="../images/comp_p3.png" alt="product3">
-            <h3 class="product-name">COMP Flavor</h3>
-            <p class="desc">A slightly sweet rich milk flavor that is delicious
-              even at carb 4.9 g / 400 kcal.</p>
-            <p class="desc">It is a nutritional food of a carbohydrate
-              adjustment model that is packed with COMP's attention to detail.</p>
-            <button class="button">
-              <i class="fas fa-shopping-cart"></i> BUY COMP Flavor
-            </button>
-          </div>
-          <div class="items-list clearfix">
-            <img class="image" src="../images/comp_p4.png" alt="product4">
-            <h3 class="product-name">COMP Ice TB v.1.0</h3>
-            <p class="desc">This time, soy milk flavored ice cream type (TB).</p>
-            <p class="desc">Reference:Japanese dietary intake standards.</p>
-            <p class="desc">(Announced by the Ministry of Health, Labor and Welfare 2020 edition)</p>
-            <button class="button">
-              <i class="fas fa-shopping-cart"></i> BUY COMP Ice TB
-            </button>
-          </div>
-        </section>
-        <section class="product-note">
-          <h6 class="note">※You may receive a product that is different from the product image posted on this site. Please note.</h6>
-        </section>
-      </main>
-    </div>
-
+    <main class="main">
+      <section class="main__nav-bar">
+        <a href="./powder_lc.html">Powder LC</a>
+        <a href="#">Powder TB</a>
+        <a href="#">Flavor</a>
+        <a href="#">Ice TB</a>
+      </section>
+      <section class="main__products-list">
```

Comment:
section = products-list
section > h2 = seo-heading--hidden
section > div = product
section > div > h3 = product__title
section > div > p = product__description
section > div > button = procuct__button

Reply:
Block is Products-list
and Block has another Blocks, products
Block products has a lot of elements, 
- product__title
- product__description
- product__button

Reply:
structure is like this

produts-list
- product
- - title
- - description
- - description
- - button
- product
- product